articles/defender-for-cloud/alerts-reference.md

```diff
@@ -310,81 +310,82 @@ Microsoft Defender for Containers provides security alerts on the cluster level
+| **Suspicious Download Then Run Activity**<br>(K8S.NODE_DownloadAndRunCombo) <sup>[1](#footnote1)</sup> | Analysis of processes running within a container or directly on a Kubernetes node, has detected a file being downloaded then run in the same command. While this isn't always malicious, this is a very common technique attackers use to get malicious files onto victim machines. | Execution, CommandAndControl, Exploitation | Medium |
+| **Digital currency mining activity**<br>(K8S.NODE_CurrencyMining) <sup>[1](#footnote1)</sup> | Analysis of DNS transactions detected digital currency mining activity. Such activity, while possibly legitimate user behavior, is frequently performed by attackers following compromise of resources. Typical related attacker activity is likely to include the download and execution of common mining tools. | Exfiltration | Low |
+| **Access to kubelet kubeconfig file detected**<br>(K8S.NODE_KubeConfigAccess) <sup>[1](#footnote1)</sup> | Analysis of processes running on a Kubernetes cluster node detected access to kubeconfig file on the host. The kubeconfig file, normally used by the Kubelet process, contains credentials to the Kubernetes cluster API server. Access to this file is often associated with attackers attempting to access those credentials, or with security scanning tools which check if the file is accessible. | CredentialAccess | Medium |
+| **Access to cloud metadata service detected**<br>(K8S.NODE_ImdsCall) <sup>[1](#footnote1)</sup> | Analysis of processes running within a container detected access to the cloud metadata service for acquiring identity token. The container doesn't normally perform such operation. While this behavior might be legitimate, attackers might use this technique to access cloud resources after gaining initial access to a running container. | CredentialAccess | Medium |
 
+<sup><a name="footnote1"></a>1</sup>: **Preview on non-AKS clusters**: This alert is generally available on AKS clusters, but on preview phase for other environments, such as Azure Arc, EKS and GKE.
 
-<sup><a name="footnote1"></a>1</sup>: **Limitations on GKE clusters**: GKE uses a Kuberenetes audit policy that doesn't support all alert types. As a result, this security alert, which is based on Kubernetes audit events, are not supported for GKE clusters.
+<sup><a name="footnote2"></a>2</sup>: **Limitations on GKE clusters**: GKE uses a Kuberenetes audit policy that doesn't support all alert types. As a result, this security alert, which is based on Kubernetes audit events, are not supported for GKE clusters.
 
-<sup><a name="footnote2"></a>2</sup>: This alert is supported on Windows nodes/containers.
+<sup><a name="footnote3"></a>3</sup>: This alert is supported on Windows nodes/containers.
 
-<sup><a name="footnote3"></a>3</sup>: Control plane alert (OS agnostic).
+<sup><a name="footnote4"></a>4</sup>: Control plane alert (OS agnostic).
```

Comment:
There is now a footnote 2,3, and 4 but no longer a footnote #1?

Shouldn't we just lower each footnote down a number so they are 1, 2 and 3? but you do have references to footnote #1   a few times here.. ex 351-361, so we still need a footnote for those

Reply:
The new one is now #1.
I've changed existing 1,2,3 to be 2,3,4 and added (new) 1 before

Reply:
Your other comment was on the new #1